Week3/homework/js-exercises/5-total-cost.js

```diff
@@ -0,0 +1,11 @@
+"use strict";
+function calculateTotalPrice(productsObject){
+    let sum=0;
+      for(let product in productObject ){
+        sum += productsObject[product];}
+    return sum;
```

Comment:
👍 for logic but again slight indentation problem here. Also, I would like to introduce to you here two new methods in JavaScript.

- `Object.keys` which you can use currently instead of a for-in loop. With modern JavaScript, it is better to make use of Object.keys instead of for-in loop. You are going to learn about the difference in JS3 when you reach inheritance.

- Also, this is a perfect use case of another method in JavaScript called **reduce** which takes in an array and reduces all the elements to a single value which in this case is our totalPrice. You will learn about this method in JS2.